cobbler/tftpgen.py

```diff
@@ -732,10 +732,10 @@ def build_kernel(self, metadata, system, profile, distro, image=None, boot_loade
             # not image based, it's something normalish
             img_path = os.path.join("/images", distro.name)
             if boot_loader in ["grub", "ipxe"]:
-                if distro.remote_boot_kernel:
-                    kernel_path = distro.remote_boot_kernel
-                if distro.remote_boot_initrd:
-                    initrd_path = distro.remote_boot_initrd
+                if distro._remote_grub_kernel:
+                    kernel_path = distro._remote_grub_kernel
+                if distro._remote_grub_initrd:
+                    initrd_path = distro._remote_grub_initrd
```

Comment:
Never ever access implementation specific attributes outside of the internal scope please! That is what we introduced the properties for. Everything with one or two underscores should be off limits outside of the original module/class.

```suggestion
                if distro.remote_grub_kernel:
                    kernel_path = distro.remote_grub_kernel
                if distro.remote_grub_initrd:
                    initrd_path = distro.remote_grub_initrd
```